pact/provider/pact-broker-loader/src/main/java/org/arquillian/algeron/pact/provider/core/loader/pactbroker/PactBrokerLoader.java

```diff
@@ -59,36 +58,32 @@
             .build();
     private Callable<HttpResponse> httpResponseCallable;
 
-    public PactBrokerLoader(final String pactBrokerHost, final String pactBrokerPort, final String pactBrokerProtocol) {
-        this(pactBrokerHost, pactBrokerPort, pactBrokerProtocol, Collections.singletonList("latest"));
+    public PactBrokerLoader() {
+        super();
```

Comment:
If you don't have any other constructor around there is no need for that. Automatically generated constructor calls `super()` by default.